data/gui/window/preferences/01_general.cfg

```diff
@@ -80,12 +80,12 @@
 			horizontal_grow = true
 			{_GUI_PREFERENCES_CHECKBOX_ALIGN_BORDER}
 			{_GUI_PREFERENCES_MAIN_COMPOSITE_SLIDER
-				( _ "Speed:")
+				( _ "Acceleration factor:")
```

Comment:
I think just "Factor:" would work.

Reply:
I think "Factor:" alone could be confusing to first timers; without reading the tooltip it's not obvious that the slider is related to the checkbox.

Reply:
That's why it's indented, actually, to indicate that. Hm...

Reply:
It's indented relative to the left edge of the checkbox but not relative to the left edge of the checkbox's text. Indent it more?

Reply:
Uh.....what? No? It aligns with the checkbox text.

Reply:
Yes, the checkbox's text ("Accelerated speed") and the slider's label ("Speed:" in master, "Factor:" or "Acceleration factor:" with this PR) are aligned. I'm proposing that the latter be shifted a bit to the right, so the word "Factor" would start under the 'l' of "Accelerated speed".

Reply:
No, that would be impossible to get right without magic numbers, nor would it look good. Not a good change.

Reply:
I didn't mean _precisely_ under the 'l', that was just an example to clarify the idea.

Reply:
Don't forget the alignment would likely be different with translations.

Reply:
If we wanted to do more indentation we'd just indent the start of the line by some constant amount. The amount wouldn't change if the text in the line above changed.

Anyway, let me know if I should change it to "Factor:".

Reply:
I think it's fine as-is, personally.